polkadot/node/core/pvf/src/host.rs

```diff
@@ -130,12 +130,33 @@ impl ValidationHost {
 			.await
 			.map_err(|_| "the inner loop hung up".to_string())
 	}
+
+	/// TEST-ONLY: resets the artifacts table.
+	#[cfg(feature = "test-utils")]
+	pub async fn prune_all_artifacts(
+		&mut self,
+		result_tx: oneshot::Sender<Result<usize, std::io::Error>>,
+	) -> Result<(), String> {
+		self.to_host_tx
+			.send(ToHost::PruneAllArtifacts { result_tx })
+			.await
+			.map_err(|_| "the inner loop hung up".to_string())
+	}
```

Comment:
Needed this method to clear the artifacts table for the benchmark. Otherwise the preparation would skip work if the artifact already existed. Could have instead deleted the artifact on-disk, and had `precheck_pvf` check if the artifact existed on-disk and not just in the artifacts table. But that seemed wrong, because in production we don't expect to ever retry prechecking.